vite.config.js

```diff
@@ -1,28 +1,258 @@
-import { defineConfig } from 'vite'
-import reactRefresh from '@vitejs/plugin-react-refresh'
-import metaversefilePlugin from 'metaversefile/plugins/rollup.js'
+/* eslint-disable no-constant-condition */
+/* eslint-disable no-unused-expressions */
+import {defineConfig} from 'vite';
+import metaversefilePlugin from 'metaversefile/plugins/rollup.js';
+import path from 'path';
+import fs from 'fs';
+import {transform} from 'esbuild';
+import glob from 'glob';
 
-// https://vitejs.dev/config/
-export default defineConfig({
-  plugins: [
-    metaversefilePlugin(),
-    reactRefresh(),
-  ],
-  optimizeDeps:{
-    entries: [
-      'src/*.js',
-      'src/*.jsx',
-      'avatars/*.js',
-      'avatars/vrarmik/*.js',
-      'src/components/*.js', 
-      'src/components/*.jsx', 
-      'src/tabs/*.jsx',
-      '*.js'
-    ],
+const esbuildLoaders = ['js', 'jsx', 'ts', 'tsx'];
+let plugins = [
+  // reactRefresh()
+];
+
+const baseDirectory = 'assets';
+const proxyModuleBase = '-proxy';
+
+let entryPoints = [
+  {
+    path: './packages/totum/index.js',
```

Comment:
These should not be hardcoded.

Reply:
All entry points are now dynamic and read from the dependencies of package.json this means we should keep only those dependencies which can be exposed to the browsers and later on used inside the metaverse-module